src/__swaps__/screens/Swap/hooks/useAssetsToSell.ts

```diff
@@ -21,7 +22,11 @@ const sortBy = (by: UserAssetFilter) => {
 };
 
 export const useAssetsToSell = () => {
-  const { accountAddress: currentAddress, nativeCurrency: currentCurrency } = useAccountSettings();
+  const { accountAddress: currentAddress, nativeCurrency: currentCurrency, network: currentNetwork } = useAccountSettings();
+
+  const provider = getCachedProviderForNetwork(currentNetwork);
+  const providerUrl = provider?.connection?.url ?? '';
+  const connectedToHardhat = isHardHat(providerUrl);
```

Comment:
I'd condense these to be the new method to do this in one line:

```
import { getIsHardhatConnected } from '@/handlers/web3';

const connectedToHardhat = getIsHardhatConnected();
```

Reply:
oops yeah